llm_bioinformatics_research/src/References/References.js

```diff
@@ -33,32 +33,40 @@ function References() {
   });
 
   return (
-    <Container sx={{ mt: 6 }}>
+    <Container className="references-container">
       <Typography variant="h4" gutterBottom>
         References
       </Typography>
       <TextField
         label="Search References"
         variant="outlined"
         fullWidth
-        style={{ marginBottom: '20px' }} 
+        className="references-search"
         margin='normal'
         color='primary'
         onChange={(e) => setSearchQuery(e.target.value)}
-      />  
-      <Paper elevation={3} style={{ padding: '20px', marginBottom: '20px' }}>
-        <Box>
-          
-          {filteredReferences.length > 0 ? (
-            filteredReferences.map((ref, index) => (
-              <li key={index}>
-                {ref.author} ({ref.year}). <a href={ref.link} target="_blank" rel="noopener noreferrer">{ref.title}</a>. {ref.journal}, {ref.volume}, {ref.pages}.
-              </li>
-            ))
-          ) : (
-            <Typography>No references found</Typography>
-          )}
-        </Box>
+      />
+      <Paper elevation={3} className="references-paper">
+        {filteredReferences.length > 0 ? (
+          filteredReferences.map((ref, index) => (
+            <Accordion key={index} className="references-accordion">
+              <AccordionSummary
+                expandIcon={<ExpandMoreIcon />}
+                aria-controls={`panel${index}-content`}
+                id={`panel${index}-header`}
+              >
+                <Typography>{ref.title}</Typography>
```

Comment:
I like how you are displaying the title. Adding the author name next to the title could be a possible update for the future if it allows for one to easily search for a reference.